components/net/http_loader.rs

```diff
@@ -892,7 +898,10 @@ fn http_network_or_cache_fetch(
         _ => {},
     }
 
-    // Step 16
+    // Step 5.15
+    // TODO: handle `Range` header
```

Comment:
Here I would change the TODO into something like "if necessary append `Accept-Encoding`/`identity` to list of headers", since "handling" the `Range` header itself is another operation that takes place inside the http-cache.